### PR TITLE
Single Weapon DPS Comparison + UnitStat Page Entry

### PR DESCRIPTION
**Pull request by chris-nos**
Single Weapon DPS Comparison + UnitStat Page Entry

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,15 +40,24 @@
...
     "firebase": "^9.17.1",
     "json-2-csv": "^3.19.0",
+    "inherits": "^2.0.4",
+    "is-glob": "^4.0.3",
+    "jsesc": "^3.0.2",
     "lodash": "^4.17.21",
     "mantine-datatable": "^1.8.6",
     "next": "^13.1.6",
     "octokit": "^2.0.14",
+    "next-offline": "^5.0.5",
```
Comment (petrvecera) [inline package.json:58]: Do we need this lib? It's notoriously outdated lib which several security holes.

File: package.json [hunk 2]
```diff
@@ -40,15 +40,24 @@
...
     "@mantine/spotlight": "^5.10.4",
+    "@nivo/bump": "^0.80.0",
+    "@nivo/core": "^0.80.0",
+    "@nivo/line": "^0.80.0",
     "@tabler/icons": "^1.106.0",
+    "chart.js": "^4.2.1",
     "dayjs": "^1.11.7",
     "firebase": "^9.17.1",
     "json-2-csv": "^3.19.0",
+    "inherits": "^2.0.4",
```
Comment (petrvecera) [inline package.json:51]: Do we need this? I think we can remove it

File: package.json [hunk 3]
```diff
@@ -40,15 +40,24 @@
     "@mantine/next": "^5.10.4",
     "@mantine/notifications": "^5.10.4",
     "@mantine/spotlight": "^5.10.4",
+    "@nivo/bump": "^0.80.0",
+    "@nivo/core": "^0.80.0",
+    "@nivo/line": "^0.80.0",
```
Comment (petrvecera) [inline package.json:45]: We can remove hte nivo if we don't usr eit 👍

File: package.json (hunk 1 above)
Comment (chris-nos) [reply package.json:58]: No, you can remove it if you want.

File: package.json (hunk 2 above)
Comment (chris-nos) [reply package.json]: yes, I also removed it in my package.json now.

File: package.json (hunk 3 above)
Comment (chris-nos) [reply package.json]: If nobody else needs it :) I removed it from my package.json.

**Pull request merged**